tox.ini

```diff
@@ -17,12 +17,13 @@ deps =
    mccabe
    pytest
    pytest-xdist
-   pyenchant
+   mypy
 
 setenv =
     COVERAGE_FILE = {toxinidir}/.coverage.{envname}
 
 commands =
+    python -Wi {envsitepackagesdir}/coverage run -m mypy --ignore-missing-imports {envsitepackagesdir}/pylint/checkers/ {posargs:}
```

Comment:
Please could this be run in a separate virtualenv to the tests. Seeing as mypy is failing at the moment, I think we should stop the mypy virtualenv running by default as well.